### ISSUE TITLE
Activating channels is raising error "SuspiciousFileOperation"

### ISSUE THREAD

**Issue by pique-b** [django/channels#1673]
After installing **channels 3.0.3**, it was added to the project's **settings.py** file in the **INSTALLED_APPS** section. Subsequently (as was required), I added the asgi dependencies as explained later.

On restarting the server and logging in the app, however, I came up with the following error in the browser:

> SuspiciousFileOperation at /audteam/
> The joined path (/audit/customize_btn_img3d.png) is located outside of the base path component (/Users/***/Desktop/proj_osh/lead/static)
> 

**Note:** **audteam** is application for managing custom users. Besides, there is another app **audit** which takes care of the main functionalities in the project **lead**.

This is how the image is being accessed (in the relevant template):

> <img class="fakeimg" src="{% static "/audit/customize_btn_img3d.png" %}" alt="No_Img"/>

If I remove either **channels** app from the **INSTALLED_APPS** section **OR** the _offending_ line (above) from the template, the issue goes away.

Why am I getting this **SuspiciousFileOperation** issue? How do I resolve this?

Here are the technical details:

OS: macOs Sierra
Python 3.6.4
Django 2.2.19
Browser Chrome (latest update)
Channels 3.0.3

**Channels setup:**
In _INSTALLED_APPS_ section
_**channels**_

Added the following reference to ASGI application:
_**ASGI_APPLICATION = 'lead.routing.application'**_

Have included **_routing.py_** in the project's main directory (where file **settings.py** resides)

In the terminal I find that the ASGI server starts up with following message:

> System check identified no issues (0 silenced).
> April 10, 2021 - 10:47:37
> Django version 2.2.19, using settings 'lead.settings'
> Starting ASGI/Channels version 3.0.3 development server at http://127.0.0.1:8000/
> Quit the server with CONTROL-C.
> 

Thanks

**Comment by carltongibson** [comment]
Try with a relative path. `audit/`

**Comment by pique-b** [comment]
Thanks for your reply.

I did that before I got your reply when I realized the mistake and of course it worked (it was basically a syntax error or more precisely, **a typo** - as I have used **relative path extensively in the particular app for serving up static files**).

However, one aspect of it bothers me. How was the app able to resolve the path (even with the error) when **channels** library had, as yet not been activated?
